docs/protocol/index.md

```diff
@@ -2,133 +2,90 @@
-
-Evmos uses [Atto](https://en.wikipedia.org/wiki/Atto-) EVMOS as the base denomination to maintain parity with Ethereum.
-
-1 evmos = 10<sup>18</sup> aevmos
-
-This matches Ethereum denomination of:
-
-1 ETH = 10<sup>18</sup> wei
-
-## Cosmos Coins
-
-Accounts can own Cosmos coins in their balance, which are used for operations with other Cosmos and transactions. Examples of these are using the coins for staking, IBC transfers, governance deposits and EVM.
-
-## EVM Tokens
-
-Evmos is compatible with ERC20 tokens and other non-fungible token standards (EIP721, EIP1155)
-that are natively supported by the EVM.
-
-## Quick Facts Table
-
-| Property               | Value                                           |
-| ---------------------- | ----------------------------------------------- |
-| Evmos Testnet          | <ProjectValue keyword="testnet_chain_id" />     |
-| Evmos Mainnet          | <ProjectValue keyword="chain_id" />             |
-| Blockchain Explorer(s) | [List of Block Explorers](./../../develop/tools/explorers) |
-| Block Time             | `~2s`                                           |
-
+- [JSON-RPC](../develop/api/ethereum-json-rpc) client for interacting with the EVM
 
+Most components are implemented in the [evm module](./modules/evm/index.md).
+To archive a seamless developer UX, however, some of the components are implemented
+outside of the module.
 
-# List of Resources
+If you want to learn more about how Evmos achieves EVM compatibility as a Cosmos chain,
+we recommend understanding the following concepts:
 
-Please find the following resources for in-depth information:
+* [Accounts](./concepts/accounts.md)
+* [Gasa and Fees](./concepts/gas-and-fees.md)
+* [Token representations](./concepts/tokens.md)
+* [Transactions](./concepts/transactions.md)
 
-- **[Networks & Connections](/develop/networks)**: List of publicly available endpoints.
-- **[Evmos Clients](/develop/build-a-dApp/clients/ethereum-JSON-RPC/clients)**: Description of available clients.
-- **[Block Explorers](/develop/build-a-dApp/tools/block-explorers)**: List of block explorers available for Evmos.
-- **[Testnet Faucet](/develop/testnet/faucet)**: Explaination of faucet use to obtain testnet tokens.
-- **Localnet**: Instructions on how to configure a local instance of the Evmos blockchain.
-  - **[Single Node](develop/build-a-dApp/run-a-node/single-node)**: Run a single local node.
-  - **[Multi Node](develop/build-a-dApp/run-a-node/multi-nodes)**: Run a local testnet with multiple nodes.
-  - **[Testnet](develop/build-a-dApp/run-a-node/testnet-commands)**: Use the testnet command of the Evmos daemon.
-- **Libraries**:
-  - **[EvmosJS](develop/build-a-dApp/tools/evmosjs)**: Javascript library for Evmos.
+## Contributing
```

Comment:
can you add a link to the contributor [guidelines](https://github.com/evmos/evmos/blob/main/CONTRIBUTING.md)?